.env

```diff
@@ -4,26 +4,26 @@
 # dev|prod
 APP_ENVIRONMENT=dev
 APP_KERNEL_SECRET=ThisTokenIsNotSoSecretChangeIt
-APP_WAIT_FOR=mysql:3306,redis:6379,rabbit-mq:5672,nchan:81
+APP_WAIT_FOR=mysql:3306,proxysql:6033,redis:6379,rabbit-mq:5672,nchan:81
```

Comment:
Communicate via unix sockets. During the load tests, ProxySQL is used as a sidecar of the app containers. The development environment should mimic this.